spatialos-sdk-tools/Cargo.toml

```diff
@@ -9,6 +9,6 @@ name = "generate_bindings"
 path = "src/generate_bindings/main.rs"
 
 [dependencies]
-clap = "2.32.0"
-glob = "0.2"
-bindgen = "0.42"
+clap = "^2"
+glob = "~0.2"
```

Comment:
Any reason for relaxing the constraints on the versions here?

Reply:
Nah not particularly. I'll revert it (as it can be done in a separate PR).